website/docs/r/imagebuilder_image_recipe.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+subcategory: "Image Builder"
+layout: "aws"
+page_title: "AWS: aws_imagebuilder_image_recipe"
+description: |-
+    Manage an Image Builder Image Recipe
+---
+
+# Resource: aws_imagebuilder_image_recipe
+
+Manages an Image Builder Image Recipe.
+
+## Example Usage
+
+```hcl
+resource "aws_imagebuilder_image_recipe" "example" {
+  block_device_mapping {
+    device_name = "/dev/xvdb"
+
+    ebs {
+      delete_on_termination = true
+      volume_size           = 100
+      volume_type           = "gp2"
+    }
+  }
+
+  component {
+    component_arn = aws_imagebuilder_component.example.arn
+  }
+
+  name         = "example"
+  parent_image = "arn:${data.aws_partition.current.partition}:imagebuilder:${data.aws_region.current.name}:aws:image/amazon-linux-2-x86/x.x.x"
+  version      = "1.0.0"
+}
+```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `component` - (Required) Configuration block(s) with components for the image recipe. Detailed below.
```

Comment:
It would be helpful for "Detailed below" to be links to the block documentation

Reply:
Similar comment stream applies (https://github.com/hashicorp/terraform-provider-aws/pull/16180#discussion_r525485952). We need a standard across the codebase (eventually all provider documentation in general) and using just the argument name (as done in a lot of documentation pages) does not generate unique links in the Terraform Registry currently.

Reply:
Created: https://github.com/hashicorp/terraform-provider-aws/issues/16407